automated install/basic-install.sh

```diff
@@ -817,12 +817,12 @@ If you want to specify a port other than 53, separate it with a hash.\
             printf -v PIHOLE_DNS_1 "%s" "${piholeDNS%%,*}"
             printf -v PIHOLE_DNS_2 "%s" "${piholeDNS##*,}"
 
-            # If the first DNS value is invalid or empty, this if statement will be true and we will set PIHOLE_DNS_1="Invalid"
-            if ! valid_ip "${PIHOLE_DNS_1}" || [[ ! "${PIHOLE_DNS_1}" ]]; then
+            # If the first DNS value is invalid (neither IPv4 nore IPv6) or empty, set PIHOLE_DNS_1="Invalid"
```

Comment:
other than the minor typo,
```suggestion
            # If the first DNS value is not a valid IPv4 or IPv6 address, or is empty, set PIHOLE_DNS_1="Invalid"
```